blockly/src/server/Server.java

```diff
@@ -1204,6 +1216,79 @@ public void move(final Direction direction) {
     vc.currentYVelocity(0);
   }
 
+  /**
+   * Moves the given entities simultaneously in a specific direction.
+   *
+   * <p>One move equals one tile.
+   *
+   * @param entity1 First entity to move
+   * @param entity2 Second entity to move
+   * @param direction Direction in which the entities will be moved.
+   */
+  public void moveSimultaneously(
```

Comment:
Aktuell ist die Methode eine Kopie von der `move` nur mit extra Logik. Es wäre besser würde die `move` einen `...` Parameter haben sodass es nur eine Methode gibt die beliebt viele Entities unterstützt.

Reply:
finde ich gut. die ellipse hat zwar auch nachteile (kann iirc z.b. auch leer sein - sollte also besser stream-mässig behandelt werden), aber damit fällt eine überladung weg und auch die für mich noch nicht verständliche beschränkung auf zwei verschiebbare objekte...

Reply:
have done that